CHANGELOG.md

```diff
@@ -36,7 +36,7 @@
 - PR #6575 Fix JNI RMM initialize with no pool allocator limit
 
 
-# cuDF 0.16.0 (Date TBD)
+# cuDF 0.16.0 (23 Oct 2020)
```

Comment:
Shouldn't it be the 21st?
https://github.com/rapidsai/cudf/blob/main/CHANGELOG.md#cudf-0160-21-oct-2020